app/build.gradle

```diff
@@ -124,7 +124,7 @@ dependencies {
 
     // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "org.jetbrains.anko:anko:0.10.5"
+    implementation "org.jetbrains.anko:anko:0.10.6"
```

Comment:
Please don't update this here. @thellmund mentioned, that there is a problem with the version in question. We have dependabot, that will create PRs for any changes in that matter.